src/net/connection/connection_pool.rs

```diff
@@ -61,6 +57,7 @@ impl ConnectionPool {
     }
 
     // Get the number of connected clients.
+    #[allow(dead_code)]
```

Comment:
Don't understand why this is dead code? if it's not exposed publicly or used internally we should remove it.